.travis/install_java_11.0.5

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+(
+  cd ~ || exit 1
+  wget "https://github.com/triplea-game/assets/raw/master/jdk-11.0.5_linux-x64_bin.tar.gz"
```

Comment:
An option would be to use the AdoptOpenJDK package, but it seems the archive variant is hosted on github as well:
https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.5%2B10/OpenJDK11U-jdk_x64_linux_hotspot_11.0.5_10.tar.gz

Might be worth trying though, maybe they have higher limits for their archives.
My other ideas:
1. Setup a simple Webserver/reuse nginx on the forum server to host the file
2. Travis has some sort of caching system, so if we download the jdk as usual and just let travis cache it, we might not hit the rate limit at all.

Reply:
```suggestion
  wget "https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.5%2B10/OpenJDK11U-jdk_x64_linux_hotspot_11.0.5_10.tar.gz"
```
I'll just go ahead and try if this works